### DISCUSSION TITLE
when exporting data to json, fields that are of type TimeStamp lose formatting when importing them to another collection. The TimeStampo Turns into a String.

### DISCUSSION THREAD

**Discussion by miguelhp373** [thanhlmm/refi-app#102] [Q&A]
when exporting data to json, fields that are of type TimeStamp lose formatting when importing them to another collection. The TimeStampo Turns into a String.

**Comment by theaspirational** [comment]
same. 
![CleanShot 2023-06-29 at 16 15 00](https://github.com/thanhlmm/refi-app/assets/48733736/2ec92329-033d-4f8b-ada0-cef5f9e6a1ab)
